src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,38 @@
+import unpaidIcon from '../../assets/images/paymentrequest_status_unpaid.svg';
+import paidIcon from '../../assets/images/paymentrequest_status_paid.svg';
+import partiallyPaidIcon from '../../assets/images/paymentrequest_status_partial.svg';
+
+interface StatusBadgeProps {
+  status: 'UNPAID' | 'PARTIAL' | 'PAID';
+}
+
+const getIconForStatus = (status: 'UNPAID' | 'PARTIAL' | 'PAID') => {
+  switch (status) {
+    case 'PARTIAL':
+      return partiallyPaidIcon;
+    case 'PAID':
+      return paidIcon;
+    case 'UNPAID':
+    default:
+      return unpaidIcon;
+  }
+};
+
+const StatusBadge = ({ status }: StatusBadgeProps) => {
+  return (
+    <div
+      className={`rounded-full py-1 px-2 space-x-2 inline-block ${
+        status === 'PAID' ? 'bg-[#D8F2EA] text-[#008055]' : 'bg-[#F1F3F4] text-[#00264D]'
```

Comment:
I def forgot to mention this before as it didn't come needed until now, sorry about that.

Anyways, I strongly suggest, from now on to use [cva](https://www.npmjs.com/package/class-variance-authority) (aka `class-variance-authority`) in these cases.

Here's a great video showing what it does and how improves the overall code: https://youtu.be/B6FrDu2Qbt0.

(if you want to go straight to the juice, you can just forward to [11:45](https://youtu.be/B6FrDu2Qbt0?t=705))